marklogic-unit-test-client/build.gradle

```diff
@@ -84,11 +84,21 @@ publishing {
   }
   repositories {
     maven {
-      name = "central"
-      url = mavenCentralUrl
-      credentials {
-        username mavenCentralUsername
-        password mavenCentralPassword
+      if (project.hasProperty("mavenUser")) {
+        name = "nexus"
+        credentials {
+          username mavenUser
+          password mavenPassword
+        }
+        url publishUrl
+        allowInsecureProtocol = true
+      }else {
```

Comment:
Should have a space after the close brace and the else here.

Reply:
Fixed